includes/Experiments.php

```diff
@@ -363,6 +363,7 @@ public function get_experiments(): array {
 				'label'       => __( 'Taxonomies', 'web-stories' ),
 				'description' => __( 'Enable support of tags and categories for stories', 'web-stories' ),
 				'group'       => 'editor',
+				'default'     => true,
```

Comment:
temp! will remove this. just trying to get to the bottom of why this new e2e test if failing remotely